setup.py

```diff
@@ -130,7 +133,7 @@ def main():
         install_requires=ldict['REQUIRES'],
         setup_requires=['future', 'configparser'],
         provides=ldict['PROVIDES'],
-        packages=find_packages(exclude=['*.tests']),
+        packages=find_packages(),
```

Comment:
It is a bit blurry in my memory now, but I think this was added to avoid tests to be run from the installed version of nipype and make sure they run from the source  I guess py.test takes care of this, so it should be fine re-adding them.